spec/requests/v0/lgy_coe_request_spec.rb

```diff
@@ -139,6 +148,34 @@
           expect(response.body).to eq @res.body
         end
       end
+
+      context 'requested document id not associated with user' do
+        before do
+          lgy_documents_response_body = [{
+            'id' => 23_929_115,
+            'document_type' => '252',
+            'create_date' => 1_670_530_715_000,
+            'description' => '[ATTACHMENT]',
+            'mime_type' => '[ATTACHMENT] example.png'
+          }, {
+            'id' => 10_101_010,
+            'document_type' => '705',
+            'create_date' => 1_670_530_714_000,
+            'description' => nil,
+            'mime_type' => 'COE Application First Returned.pdf'
+          }]
+          lgy_documents_response = double(:lgy_documents_response, body: lgy_documents_response_body)
+          expect_any_instance_of(LGY::Service).to receive(:get_coe_documents).and_return(lgy_documents_response)
+        end
+
+        it '404s' do
+          # Note that this ID is not present in lgy_documents_response_body above.
```

Comment:
Maybe one more quick test for one that DOES exist, but that does not belong to user?

Reply:
It was proving more difficult than it was worth to write a test like this, so I'm going to skip it. I think that's okay here because whether a document exists in LGY or not is not our concern. We're only concerned with whether the requested document id was returned in the documents request for this particular user.